conda-requirements.txt

```diff
@@ -34,3 +34,6 @@ nc_time_axis
 pandas
 python-stratify
 pyugrid
+
+# Iris extensions (i.e. key tools that depend on Iris)
```

Comment:
Defines a concept of "Iris extensions"
This relates to where we have struggled with migrating grib-code to dask, circular dependencies etc.

What we found is that these :
  * depend on Iris + import it
  * only use public Iris API
  * are *not* included by Iris, except dynamically to support specific functions
  * can use Iris in their tests
  * are not tested in Iris
    * except integration tests on Iris-terms results (e.g. cube properties)
    * definitely not details of the extension within Iris tests